core/services/major_tom/major_tom.py

```diff
@@ -0,0 +1,82 @@
+import asyncio
+from typing import Any, Dict, List
+
+import requests
+from commonwealth.settings.manager import Manager
+from loguru import logger
+
+from session_tracker import SessionTracker
+from settings import SettingsV1
+from system_data import SystemDataGatherer
+
+SERVICE_NAME = "major_tom"
+TELEMETRY_VERSION = 1
+
+
+class MajorTom:
+    """
+    Consolidates data from SessionTracker and SystenDataGatherer in one place and sends it to GroundControl
+    """
+
+    last_session = Dict[str, Any]
+
+    def __init__(self) -> None:
+        self.system_data_gatherer = SystemDataGatherer()
+        self.settings_manager = Manager(SERVICE_NAME, SettingsV1)
+        self.sessions_folder = self.settings_manager.config_folder / "sessions"
+        self.sessions_folder.mkdir(parents=True, exist_ok=True)
+        self.session_tracker = SessionTracker(self.sessions_folder)
+        self.settings = self.settings_manager.settings
+        self.remote = self.settings.remote
+
+    def phoneHome(self, vehicle_data: Dict[str, Any], system_data: Dict[str, Any]) -> int:
```

Comment:
We should have a better structure than a dict to ensure that we are not going to break the API shared with ground control.